### ISSUE TITLE
MASK RCNN training on own data

### ISSUE THREAD

**Issue by shafique18** [matterport/Mask_RCNN#1230]
Hi,
As per your description I'm trying to implement a customize model to detect number strip in credit cards which are available in google.
I collected around 200 images from google and after augmentation I have around 700 images.
I trained model as per your instruction but after training I got only one correct prediction out of ten.
could you please help me how I can increase my model accuracy.

Details are :- 
machine = 11 GB RAM GPU
Batch_size = 2
epoch = 100
steps per epoch = 700
Training time = 13 hours
Backend model weights - 
                              - mask_rcnn_coco.h5
                              - resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5

Please help me to increase my model accuracy or help me to create a customize  model which will help me to understand how a mask RCNN model is working.

**Comment by stygian2a** [comment]
Can we see some training/validation losses curves?

**Comment by shafique18** [comment]
loss: 0.1825 
rpn_class_loss: 0.0013
rpn_bbox_loss: 0.0421
mrcnn_class_loss: 0.0055
mrcnn_bbox_loss: 0.0242
mrcnn_mask_loss: 0.1095
val_loss: 0.6257
val_rpn_class_loss: 0.0028
val_rpn_bbox_loss: 0.3404
val_mrcnn_class_loss: 0.0093
val_mrcnn_bbox_loss:0.1108
val_mrcnn_mask_loss: 0.1625

**Comment by shafique18** [comment]
loss: 0.478
rpn_class_loss: 2.6125e-04
rpn_bbox_loss: 0.0101
mrcnn_class_loss: 0.016
mrcnn_bbox_loss: 0.0030
mrcnn_mask_loss: 0.0329
val_loss: 0.7351
val_rpn_class_loss: 0.0023
val_rpn_bbox_loss: 0.3861
val_mrcnn_class_loss: 0.0094
val_mrcnn_bbox_loss:0.0800
val_mrcnn_mask_loss: 0.2473

**Comment by shafique18** [comment]
> Can we see some training/validation losses curves?

**Comment by ApoorvaSuresh** [comment]
hi, I've a questions, how can you specify weights for the resnet? 

and when we specify coco weights, to which network is it alotted?
